implementations/README.md

```diff
@@ -77,8 +77,6 @@ Despite the fact that an `ERC725Account` is an **ownable contract**, the selecto
 ^ isValidSignature     // ERC1271
 ```
 
```

Comment:
Maybe you can add the final value of this interface id? Like `0xXXXX` ?

Reply:
@Hugoo done ✅